src/components/calcite-slider/calcite-slider.scss

```diff
@@ -34,6 +34,10 @@
   font-size: var(--calcite-slider-container-font-size);
 }
 
+.handle__label--flex {
```

Comment:
Could we name this something better to reflect its purpose?

Reply:
may be  `hyphen--wrap` ?